src/AVFoundation/AVCaptureReactionType.rgen.cs

```diff
@@ -0,0 +1,44 @@
+#pragma warning disable APL0003
+using System;
+using System.Runtime.Versioning;
+
+using Foundation;
+using ObjCRuntime;
+using ObjCBindings;
+
+#nullable enable
+
+namespace AVFoundation {
+
+	[BindingTypeAttribute]
+	[SupportedOSPlatform ("ios17.0")]
+	[SupportedOSPlatform ("tvos17.0")]
+	[SupportedOSPlatform ("maccatalyst17.0")]
+	[SupportedOSPlatform ("macos14.0")]
+	public enum AVCaptureReactionType {
+		[Field ("AVCaptureReactionTypeThumbsUp")]
```

Comment:
RGen doesn't do all it's supposed: the [Field] attributes are supposed to generate a bunch of additional code, which isn't happening.

If that's coming later, then the [Field] attribute should be removed now, and added back when the [Field] support is back.